sdcm/remote/kubernetes_cmd_runner.py

```diff
@@ -213,8 +215,9 @@ def receive_files(self, src, dst, delete_dst=False, preserve_perm=True, preserve
     # pylint: disable=too-many-arguments,unused-argument
     @retrying(n=3, sleep_time=5, allowed_exceptions=(RetryableNetworkException, ))
     def send_files(self, src, dst, delete_dst=False, preserve_symlinks=False, verbose=False):
-        KubernetesOps.copy_file(self.kluster, src, f"{self.namespace}/{self.pod_name}:{dst}",
-                                container=self.container, timeout=300)
+        with KEY_BASED_LOCKS.get_lock(f"k8s--{self.kluster.name}--{self.namespace}--{self.pod_name}"):
```

Comment:
I think we should have a property for the lock name

Reply:
Why? It is going to be the only place of it's usage. At least for upcoming future.

Reply:
i.e. if we gonna clean that lock from the mapping

Reply:
If we have more usages, yes. But key deletion is also not really needed. At least, now.